articles/cli.md

```diff
@@ -0,0 +1,178 @@
+## Bonsai Command-Line Interface (CLI)
+
+Bonsai CLI (Command-Line Interface) allows workflows to be launched from the OS command-line tool. In addition to opening workflows with the default graphic interface, the CLI opens the possibility to use other advanced features, that will be covered ahead, such as "no-editor" mode, initialization of variable's values, and specifying layouts.
+
+
+&nbsp;
+___
+### Installation and basic use
+Bonsai CLI can be called by targeting ```Bonsai.exe``` executable. To make sure you are able to call ```Bonsai``` from the command line:
```

Comment:
Another small suggestion for greater clarity: 

"The Bonsai CLI can be called by targeting the ```Bonsai.exe``` executable. To make sure you are able to call ```Bonsai``` from the command line, add it to your path using the following command:"